### PR TITLE
Revamped the mgmt instance call.

### PR DESCRIPTION
**Pull request by TimSimpsonR**

### CODE REVIEW

File: reddwarf/extensions/mgmt/instances/models.py [hunk 1]
```diff
@@ -72,25 +76,34 @@ def deleted(self):
...
-            assert instance.server is not None
-
-        except Exception as e:
-            LOG.error(e)
-            instance = load_instance(cls, context, id, needs_server=False)
+            instance.volume = client.volumes.get(instance.volume_id)
+        except Exception as ex:
+            instance.volume = None
+        for d in dir(instance.volume):
+            print(str(d) + "=" + str(getattr(instance.volume, d)))
```
Comment (ed-) [inline reddwarf/extensions/mgmt/instances/models.py]: Maybe this should be `LOG.debug`?

File: reddwarf/tests/fakes/nova.py [hunk 2]
```diff
@@ -346,15 +351,24 @@ def __init__(self, parent, owner, id, size, display_name,
...
         return ("FakeVolume(id=%s, size=%s, "
-               "display_name=%s, display_description=%s)") % (self.id,
-               self.size, self.display_name, self.display_description)
+               "display_name=%s, display_description=%s, _current_status=%s)"
+               % (self.id, self.size, self.display_name,
+                  self.display_description, self._current_status))
+
+    @property
+    def availability_zone(self):
+        return "fake-availability-zone"
```
Comment (ed-) [inline reddwarf/tests/fakes/nova.py:367]:
``` text
SONIC CLEARED ACT 1
- Fake Availability Zone -
```

File: reddwarf/tests/fakes/nova.py [hunk 3]
```diff
@@ -346,15 +351,24 @@ def __init__(self, parent, owner, id, size, display_name,
...
+
+    @property
+    def created_at(self):
+        return "2001-01-01-12:30:30"
```
Comment (ed-) [inline reddwarf/tests/fakes/nova.py:371]: Man, that's expensive.

File: reddwarf/extensions/mgmt/instances/models.py (hunk 1 above)
Comment (TimSimpsonR) [inline reddwarf/extensions/mgmt/instances/models.py]: Indeed!

**Comment by ed-** [comment]
+1

File: reddwarf/tests/fakes/nova.py (hunk 2 above)
Comment (cp16net) [inline reddwarf/tests/fakes/nova.py:367]: lol

**Comment by jcru** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**